components/prism-ignore.js

```diff
@@ -0,0 +1,34 @@
+(function (Prism) {
+	Prism.languages.ignore = {
+		// https://git-scm.com/docs/gitignore
+		'header': {
+			pattern: /(^|[^\\])###.*/,
+			lookbehind: true,
+			alias: ['comment', 'bold']
+		},
+		'section': {
+			pattern: /(^|[^\\])##.*/,
+			lookbehind: true,
+			alias: 'comment'
+		},
+		'comment': {
+			pattern: /(^|[^\\])#.*/,
+			lookbehind: true,
+			alias: 'namespace'
+		},
+		'entry': {
+			pattern: /\S(?:.*(?:(?:\\ )|\S))?/,
+			alias: 'string',
+			inside: {
+				'regex': {
+					pattern: /^!|(^|[^\\])[\[\]]/,
+					lookbehind: true
+				},
```

Comment:
I think the `!` should be highlighted as an operator.

Also, instead of highlighting the `[]` delimiters of regexes, wouldn't it be more useful to highlight the whole regex?

```suggestion
			inside: {
				'operator': /^!/,
				'regex': {
					pattern: /(^|[^\\])\[[^\[\]]*\]/,
					lookbehind: true
				},
```

Reply:
Yes, that looks better